spec/dummy_app/config/application.rb

```diff
@@ -31,7 +31,12 @@ class Application < Rails::Application
       if v >= Gem::Version.new("4.2") && v < Gem::Version.new("5.0.0.beta1")
         config.active_record.raise_in_transactional_callbacks = true
       end
-      if v >= Gem::Version.new("5.0.0.beta1")
+      if v >= Gem::Version.new("5.0.0.beta1") && v < Gem::Version.new("5.1")
+        config.active_record.belongs_to_required_by_default = true
```

Comment:
Why not `load_defaults("5.0")`?

Reply:
Apparently this command was added in Rails 5.1.